templates/compose/gateway.yaml.erb

```diff
@@ -8,10 +8,11 @@ services:
       - ../config/gateway:/etc/envoy/
     command: /usr/local/bin/envoy -l info -c /etc/envoy/envoy.yaml
     labels:
-      traefik.enable: true
-      traefik.frontend.rule: "PathPrefix:/api/, /admin, /assets/;Host:<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>"
-      traefik.default.port: 8099
-      traefik.default.protocol: http
-      traefik.wss.frontend.rule: "Host:<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>"
-      traefik.wss.protocol: ws
-      traefik.wss.port: 8099
+      - "traefik.http.routers.gateway.rule=Host(`<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>`) && PathPrefix(`/api`,`/admin`,`/assets/`)"
```

Comment:
I think /api, /assets and /admin are missing here

Reply:
No

Reply:
It works fine